Commands/Monitor.php

```diff
@@ -35,6 +36,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $systemCheck->checkRedisIsInstalled();
         }
 
+        $iterations = $input->getOption('iterations');
+        if (empty($iterations) && $iterations !== 0 && $iterations !== '0') {
```

Comment:
Maybe you could extract this into a method to keep the main command a bit small? Like 

`$this->getNumInteractions($input, $output);
private function getNumInteractions($input, $output) {}
`